axelrod/strategies/_strategies.py

```diff
@@ -105,6 +106,7 @@
     LimitedRetaliate,
     LimitedRetaliate2,
     LimitedRetaliate3,
+    EvolvedLookerUp,
```

Comment:
You need to also add `LookerUp` to the list of strategies. (So that it's easily accessible).